gui/default/syncthing/core/syncthingController.js

```diff
@@ -780,6 +780,60 @@ angular.module('syncthing.core')
             return 'info';
         };
 
+        $scope.syncthingStatus = function () {
+            
+            var deviceCount = $scope.devices.length;
+            var syncCount = 0;
+            var notifyCount = 0;
+            var pauseCount = 0;
+            
+            //loop through all devices
+            for (var i = 0; i < $scope.devices.length; i++) {
+                var status = $scope.deviceStatus({
+                    deviceID:$scope.devices[i].deviceID
+                });
+                switch (status) {
+                    case 'syncing': syncCount++; break;
+                    case 'unknown': notifyCount++; break;
+                    case 'paused': pauseCount++; break;
+                    case 'unused': deviceCount--; break;
+                }
+            }
+
+            //loop through all folders
+            var folderListCache = $scope.folderList();
+            for (var i = 0; i < folderListCache.length; i++) {
+                var status = $scope.folderStatus(folderListCache[i]);
+                switch (status) {
+                    case 'syncing':
+                    case 'scanning':     //possible?
+                        syncCount++;
+                        break;
```

Comment:
so just for syncing? i viewed it more as an activity icon... for some users the scanning takes a long time so thought it would be nice to include scanning.